File: __tests__/__fixtures__/output/ruby/native/cookies.rb
require 'uri'
require 'net/http'
require 'openssl'

url = URI("https://httpbin.org/cookies")

http = Net::HTTP.new(url.host, url.port)
http.use_ssl = true

request = Net::HTTP::Get.new(url)
request["cookie"] = 'foo=bar; bar=baz'

response = http.request(request)
puts response.read_body

File: src/fixtures/requests/cookies.js
module.exports = {
  log: {
    version: '1.2',
    creator: {
      name: 'HTTPSnippet',
      version: '1.0.0',
    },
    entries: [
      {
        request: {
          method: 'GET',
          url: 'https://httpbin.org/cookies',
          cookies: [
            {
              name: 'foo',
              value: 'bar',
            },
            {
              name: 'bar',
              value: 'baz',
            },
          ],
        },
        response: {
          status: 200,
          statusText: 'OK',
          httpVersion: 'HTTP/1.1',
          headers: [
            {
              name: 'Content-Type',
              value: 'application/json',
            },
          ],
          content: {
            size: -1,
            mimeType: 'application/json',
            text: JSON.stringify({
              bar: 'baz',
              foo: 'bar',
            }),
          },
          headersSize: -1,
          bodySize: -1,
        },
      },
    ],
  },
};
